python_scripts/parameter_tuning_grid_search.py

```diff
@@ -49,14 +49,9 @@
 # We will define a pipeline as seen in the first module. It will handle both
 # numerical and categorical features.
 #
-# As we will use a tree-based model as a predictor, here we apply an ordinal
-# encoder on the categorical features: it encodes every category with an
-# arbitrary integer. For simple models such as linear models, a one-hot encoder
-# should be preferred. But for complex models, in particular tree-based models,
-# the ordinal encoder is useful as it avoids having high-dimensional
-# representations.
-#
-# First we select all the categorical columns.
+# Here we will use a tree-based model as a predictor. That means that only the
```

Comment:
Maybe this would be worth simplifying this part and the part below to only say something about tree-based models (and not mention linear models since we are not using them in this notebook):
- they don't need scaling for the numerical variables
- they can deal with an `OrdinalEncoder` even if the integer coding order is not meaningful